tools/build_variables.bzl

```diff
@@ -1231,6 +1231,7 @@ aten_native_source_non_codegen_list = [
     "aten/src/ATen/native/quantized/cpu/qmatmul.cpp",
     "aten/src/ATen/native/quantized/cpu/qmul.cpp",
     "aten/src/ATen/native/quantized/cpu/qnormalization.cpp",
+    "aten/src/ATen/native/quantized/cpu/qpixel_shuffle.cpp",
```

Comment:
another nit: we are trying to aligning the filename under native/quantized/cpu to be the same as the floating point variant, e.g.: https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/cpu/PixelShuffleKernel.cpp can you change the name as well